website/client/components/header/menu.vue

```diff
@@ -380,10 +380,7 @@ export default {
       modalStack: 'modalStack',
     }),
     navbarZIndexClass () {
-      if (this.modalStack.length > 0) {
-        return 'navbar-z-index-modal';
-      }
-      return 'navbar-z-index-normal';
+      return 'navbar-z-index-modal';
```

Comment:
@negue or @SabreCat I think these two different classes had a reasoning, do you remember why?

Reply:
The css code is 

```
  .navbar-z-index {
    &-normal {
      z-index: 1080;
    }

    &-modal {
      z-index: 1035;
    z-index: 1042; // To stay above snakbar notifications and modals
    }
  }
```

not sure why the two z-indexes on the second class

Reply:
That line was removed a few months ago: https://github.com/HabitRPG/habitica/commit/6c64a1cd8c48244ccbf9d971e75aedbf16c0d1c8

Looks like it was accidentally re-added here--mistake in merge conflict resolution perhaps? https://github.com/HabitRPG/habitica/commit/ba52a90d93edfa023eba0c7c4f68dab76443c3fe

Reply:
So it looks like the entire issue here is that in some cases the modalStack isn't updated correctly, this fix it but I fear it'll break when a normal modal is shown. Is there a reason we needed both css rules?

Reply:
We can try it and test. It's been long enough I have no recollection, sorry...

Reply:
@wideawake42 let's give this a try, can you please remove the `navbar-z-index-modal` css rule now that it's not used anymore? then I'll merge it

Reply:
Sure! Will do -- it will be later tonight though.

Reply:
Wait, do you mean the second rule that was accidentally re-added?

Reply:
Sorry I meant these two lines https://github.com/HabitRPG/habitica/blob/develop/website/client/components/header/menu.vue#L197-L200